service/views.py

```diff
@@ -22,3 +23,16 @@ def service_list(request):
     }
 
     return render(request, 'service_list.html', context)
+
+def service_update(request, service_id):
+    service = get_object_or_404(Service, id=service_id)
+    if request.method == "POST":
+        form = CreateNewService(request.POST, instance=service)
+        if form.is_valid():
+            form.save()
+            return redirect('service_list')
+        else:
+            messages.error(request, 'Formulario con errores')
```

Comment:
There is no validation for dates. This update form allows future dates, and there is no conexion between payment and service. You may ask if validations is needed in this model